### PR TITLE
237 feature update CRP api endpoints to use database

### PR DESCRIPTION
**Pull request by sandeep-ps**
This PR updates the CRP state-distribution and summary endpoints to use the database. 

The endpoints are:

1. `GET /pdl/titles/title-ii/programs/crp/summary`
2. `GET /pdl/titles/title-ii/programs/crp/state-distribution`

Please review after doing local data ingestion using this PR https://github.com/policy-design-lab/data-ingestion/pull/30.

### CODE REVIEW

**Comment by sandeep-ps** [comment]
New CRP Summary Response:

```
{
    "totalPaymentInDollars": 8945384000.0,
    "totalRecipients": null,
    "totalFarms": 1658495,
    "totalContracts": 2941626,
    "totalAreaInAcres": 109366879,
    "statutes": [],
    "subPrograms": [
        {
            "subProgramName": "Continuous Sign-up",
            "totalPaymentInDollars": 5545998000.0,
            "totalPaymentInPercentage": 23.66,
            "subSubPrograms": [
                {
                    "subSubProgramName": "Continuous Non-CREP",
                    "totalPaymentInDollars": 4468753000.0,
                    "totalPaymentInPercentage": 80.58,
                    "totalRecipients": null,
                    "totalFarms": 1175922,
                    "totalContracts": 1775211,
                    "totalAreaInAcres": 32966061
                },
                {
                    "subSubProgramName": "CREP Only",
                    "totalPaymentInDollars": 774709000.0,
                    "totalPaymentInPercentage": 13.97,
                    "totalRecipients": null,
                    "totalFarms": 185298,
                    "totalContracts": 272549,
                    "totalAreaInAcres": 4503059
                },
                {
                    "subSubProgramName": "Farmable Wetland",
                    "totalPaymentInDollars": 304190000.0,
                    "totalPaymentInPercentage": 5.48,
                    "totalRecipients": null,
                    "totalFarms": 56640,
                    "totalContracts": 73338,
                    "totalAreaInAcres": 1925576
                }
            ],
            "totalRecipients": null,
            "totalFarms": 1223725,
            "totalContracts": 2120933,
            "totalAreaInAcres": 39393553
        },
        {
            "subProgramName": "General Sign-up",
            "totalPaymentInDollars": 3282560000.0,
            "totalPaymentInPercentage": 14.01,
            "subSubPrograms": [],
            "totalRecipients": null,
            "totalFarms": 576541,
            "totalContracts": 793959,
            "totalAreaInAcres": 61886172
        },
        {
            "subProgramName": "Grassland",
            "totalPaymentInDollars": 116822000.0,
            "totalPaymentInPercentage": 0.5,
            "subSubPrograms": [],
            "totalRecipients": null,
            "totalFarms": 15836,
            "totalContracts": 26773,
            "totalAreaInAcres": 8087510
        }
    ]
}

I will deploy this to the development instance soon.
```

**Comment by pengyin-shan** [comment]
> Hi, @pengyin-shan, CRP data is now ingested in the development database, and the API deployed as well. Please look at the JSON response and let me know if you have any questions.
> 
> A change that I had previously done and pushed, but I forgot to mention during today's meeting - The word "Total" has been removed from the sub-program names. If for display purposes, total has to be added back, please do so. The actual sub-program name does not contain "Total". I think it was probably added to indicate that these were the total amount.

Thank you @sandeep-ps ! Looks good to me, I will start the CRP page development then.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,20 +4,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
```
Comment (pengyin-shan) [inline CHANGELOG.md:10]: I'd suggest removing this empty line since other titles and context are together

**Review by pengyin-shan — APPROVED** [review]
Looks good, just add a comment for the CHANGELOG format. Approving this PR.

File: CHANGELOG.md (hunk 1 above)
Comment (sandeep-ps) [reply CHANGELOG.md]: Thanks. Updated.

**Comment by sandeep-ps** [comment]
I accidentally pushed ACEP and RCPP changes to this PR. I have reverted those commits. Now, this contains only CRP changes.

File: CHANGELOG.md (hunk 1 above)
Comment (sandeep-ps) [reply CHANGELOG.md]: Resolving this conversation.

**Review by ywkim312 — APPROVED** [review]

**Pull request merged**